caddy.go

```diff
@@ -836,7 +839,7 @@ var (
 	instancesMu sync.Mutex
 )
 
-const (
+var (
```

Comment:
Remember that Caddy can (soon) be compiled with multiple server types built in - the HTTP server will look for Caddyfile by default but looking for Corefile in that case just because it's built with the DNS server plugged in would be confusing. Instead of changing this, have your server type specify default Caddyfile contents.